inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -684,7 +684,44 @@
         },
         "references": {
             "items": {
-                "$ref": "elements/reference.json"
+                "properties": {
+                    "curated_relation": {
+                        "type": "boolean"
+                    },
+                    "raw_refs": {
```

Comment:
References are an aggregated list of raw references (e.g. say that we are putting together arXiv and publisher)